app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+        self.income = 0
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_list: list) -> float:
+        for car in cars_list:
+            if car.clean_mark < self.clean_power:
+                self.income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return round(self.income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        washing_price = car.comfort_class * \
+            (self.clean_power - car.clean_mark) * \
+            self.average_rating / self.distance_from_city_center
```

Comment:
That's bad example of splitting some long math, try to get rid of "\" at the end of the line
Good example:

how_much_fuel_needed = (max_fuel_consumption
                        * height_fuel_consumption_coeficient
                        * distance_to_the_destinatoin
                        / (plan_max_speed
                           - wind_awerage_speed
                           * wind_angle_coefisient)
                        * overlap_coeficient)
Bad example:

how_much_fuel_needed = max_fuel_consumption \
                       * height_fuel_consumption_coeficient \
                       * distance_to_the_destinatoin / (
                               plan_max_speed 
                               - wind_awerage_speed 
                               * wind_angle_coefisient
                       ) * overlap_coeficient